keps/sig-node/4328-node-label-manager/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: "Node Label Manager"
+kep-number: 4328
+authors:
+  - "@AxeZhan"
+owning-sig: sig-node
```

Comment:
Given the similarity with the taint eviction, I agree that it should be owned by sig-scheduling.